### DISCUSSION TITLE
Trying to run simulation model from a research paper

### DISCUSSION THREAD

**Discussion by datboigbenga** [topas-nbio/TOPAS-nBio#41] [How to?]
<img width="505" alt="Screenshot 2024-10-11 at 1 15 45 PM" src="https://github.com/user-attachments/assets/49f273b1-15bc-4d5b-83e3-a1149012f1ee">
<img width="196" alt="Screenshot 2024-10-11 at 1 16 10 PM" src="https://github.com/user-attachments/assets/43b79dc5-27ad-4271-9565-d7e186e53570">

Kindly assist me in writing topas code for this research methodology. Thanks

**Comment by jschuemann** [comment]
Hi,
we are happy to help with TOPAS or TOPAS-nBio related questions when we can and find the time, but we are not a code writing service. But, if you write an example for this, you are welcome to share it here for others to use it, too.
Best
     Jan
